contrib/txr/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "txr"
+pkgver = "293"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--parallelmake", "--prefix=/usr"]
+make_cmd = "gmake"
+make_check_target = "tests"
+hostmakedepends = ["bash", "gmake"]
+makedepends = ["libffi-devel", "zlib-devel"]
+pkgdesc = "Data munging language"
+maintainer = "Paul A. Patience <paul@apatience.com>"
+license = "custom:txr"
+url = "https://www.nongnu.org/txr"
+source = f"https://www.kylheku.com/cgit/{pkgname}/snapshot/{pkgname}-{pkgver}.tar.bz2"
+sha256 = "6fc21ae7332f98f97af35ad3ca1808d0043c4c85384c4e7bebcfce967e36fa5c"
+hardening = ["vis"]
+options = ["!cross", "!lto"]
```

Comment:
is this actually !cross and !lto?

Reply:
Yes. LTO causes issues with the GC; some tests fail. As for cross, TXR builds its stdlib by invoking the `txr` binary, so if I read `Packaging.md` correctly, it is not eligible for cross-compilation.

Reply:
sounds good :)